src/redux/books/Books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] It would be better if you write your action types following the [ducks pattern](https://github.com/erikras/ducks-modular-redux). 

```suggestion
const ADD_BOOK = 'bookstore/books/ADD_BOOK';
const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
```